src/scripts/app.js

```diff
@@ -1 +1,22 @@
 console.log('hello');
+
+Array.from(document.querySelectorAll('.stars')).forEach(function (a) {
+  a.addEventListener('click', function (e) {
+    e.preventDefault();
+    console.log(e);
+    e.originalTarget.classList.add('full');
```

Comment:
`e.originalTarget ` nie jest wspierane tylko na FireFox?

Reply:
`originalTarget` is Non-standard please use `target` or `currentTarget`.

Reply:
poprawione